docs/integration/push-notification.adoc

```diff
@@ -96,9 +96,14 @@ The header includes the basic information about the message   and equals the mes
 |7 |sequence_number |int64 |The sequence number to determine the correct order for messages, that were recorded at the same time point
 |8 |sender_id |string |The endpoint ID of the sender
 |9 |created_at |Timestamp |The timestamp, when this message was added to the receiving endpoints feed
+|10 | message_id |String | Internal AgriRouter message ID representing this message and its payload
+|11 | metadata | agrirouter.commons.Metadata| Additional metadata information to help differentiate between messages of the same type 
```

Comment:
See former comment.

Reply:
I need clarification here; agrirouter instead of AgriRouter or what's the issue?